### PR TITLE
✨ feat: migrate less to token for Tabs

### PR DESCRIPTION
**Pull request by heiyu4585**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     migrate less to token for Tabs      |
| 🇨🇳 Chinese |     迁移tabs less token      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at ab95c0d</samp>

Refactored the tabs component styles interface in `index.ts` to improve readability and extensibility.

### <samp>🤖 Generated by Copilot at ab95c0d</samp>

*  Reorder import statements and interface properties alphabetically for readability and consistency ([link](https://github.com/ant-design/ant-design/pull/42186/files?diff=unified&w=0#diff-d2d317a91ba62030532018942e2f2e91ffc533b238fb8017b7bea62d4ac428a7L2-R18))
*  Replace redundant `tabsCardHeight` variable with `token.controlHeightLG` value in `tabsCardHorizontalPadding` property ([link](https://github.com/ant-design/ant-design/pull/42186/files?diff=unified&w=0#diff-d2d317a91ba62030532018942e2f2e91ffc533b238fb8017b7bea62d4ac428a7L871-L872))
*  Move `tabsCardHeight` and `tabsCardHeadBackground` properties to the end of the interface to group them with other card-related properties ([link](https://github.com/ant-design/ant-design/pull/42186/files?diff=unified&w=0#diff-d2d317a91ba62030532018942e2f2e91ffc533b238fb8017b7bea62d4ac428a7L878-R881))
*  Add commented-out properties to the interface as placeholders for future implementation of tabs component styles based on Less variables ([link](https://github.com/ant-design/ant-design/pull/42186/files?diff=unified&w=0#diff-d2d317a91ba62030532018942e2f2e91ffc533b238fb8017b7bea62d4ac428a7R900-R923))

### CODE REVIEW

File: docs/react/migrate-less-variables.zh-CN.md [hunk 1]
```diff
@@ -179,7 +179,36 @@ title: Less 变量迁移 Design Token
 
 <!-- ### Table 表格 -->
 
-<!-- ### Tabs 标签页 -->
+### Tabs 标签页
+
+<!-- prettier-ignore -->
+| Less 变量 | Component Token | 备注 |
+| --- | --- | --- |
+| `@tabs-card-head-background` | `cardHeadBackground` | - |
```
Comment (MadCcc) [inline docs/react/migrate-less-variables.zh-CN.md:187]:
```suggestion
| `@tabs-card-head-background` | `cardHeaderBg` | - |
```

File: docs/react/migrate-less-variables.zh-CN.md [hunk 2]
```diff
@@ -179,7 +179,36 @@ title: Less 变量迁移 Design Token
...
+| `@tabs-card-heigh` | `cardHeight` | - |
```
Comment (MadCcc) [inline docs/react/migrate-less-variables.zh-CN.md:188]:
```suggestion
| `@tabs-card-height` | `cardHeight` | - |
```

File: docs/react/migrate-less-variables.zh-CN.md [hunk 3]
```diff
@@ -179,7 +179,36 @@ title: Less 变量迁移 Design Token
...
+| `@tabs-card-active-color` | `cardActiveColor` | - |
+| `@tabs-card-horizontal-padding` | `cardHorizontalPadding` | - |
+| `@tabs-card-horizontal-padding-sm` | `cardHorizontalPaddingSm` | - |
+| `@tabs-card-horizontal-padding-lg` | `cardHorizontalPaddingLg` | - |
```
Comment (MadCcc) [inline docs/react/migrate-less-variables.zh-CN.md:192]: horizontal-padding 都可以换成 paddingInline，用的地方

File: docs/react/migrate-less-variables.zh-CN.md [hunk 4]
```diff
@@ -179,7 +179,36 @@ title: Less 变量迁移 Design Token
...
+| `@tabs-title-font-size` | `titleFontSize` | - |
+| `@tabs-title-font-size-lg` | `titleFontSizeLg` | - |
+| `@tabs-title-font-size-sm` | `titleFontSizeSm` | - |
+| `@tabs-ink-bar-color` | `inkBarColor` | - |
+| `@tabs-bar-margin` | `barMargin` | - |
+| `@tabs-horizontal-gutter` | `horizontalGutter` | - |
+| `@tabs-horizontal-margin` | `horizontalMargin` | - |
+| `@tabs-horizontal-margin-rtl` | `horizontalMarginRtl` | - |
```
Comment (MadCcc) [inline docs/react/migrate-less-variables.zh-CN.md:200]: 这个应该可以废弃

File: components/tabs/style/index.ts [hunk 5]
```diff
@@ -898,6 +909,31 @@ export default genComponentStyleHook(
...
+    titleFontSize: token.fontSize,
+    titleFontSizeLg: token.fontSizeLG,
+    titleFontSizeSm: token.fontSizeSM,
+    inkBarColor: token.colorPrimary,
+    barMargin: `0 0 ${token.marginMD} 0`,
+    horizontalGutter: 32, // Fixed Value
+    horizontalMargin: '0 0 0 32px', // Fixed Value
+    horizontalPadding: `${token.paddingSM} 0`,
+    horizontalPaddingLg: `${token.paddingMD} 0`,
+    horizontalPaddingSm: `${token.paddingXS} 0`,
```
[the remark this answers is not in the document]
Comment (heiyu4585) [reply components/tabs/style/index.ts:928]: done

File: docs/react/migrate-less-variables.zh-CN.md [hunk 6]
```diff
@@ -298,7 +298,36 @@ title: Less 变量迁移 Design Token
...
+| --- | --- | --- |
+| `@tabs-card-head-background` | `cardBg` | - |
+| `@tabs-card-height` | `cardHeight` | - |
+| `@tabs-card-active-color` | `cardActiveColor` | - |
+| `@tabs-card-horizontal-padding` | `cardHorizontalPadding` | - |
+| `@tabs-card-horizontal-padding-sm` | `cardHorizontalPaddingSm` | - |
+| `@tabs-card-horizontal-padding-lg` | `cardHorizontalPaddingLg` | - |
+| `@tabs-title-font-size` | `titleFontSize` | - |
+| `@tabs-title-font-size-lg` | `titleFontSizeLg` | - |
+| `@tabs-title-font-size-sm` | `titleFontSizeSm` | - |
```
Comment (MadCcc) [inline docs/react/migrate-less-variables.zh-CN.md:314]:
```suggestion
| `@tabs-title-font-size` | `titleFontSize` | - |
| `@tabs-title-font-size-lg` | `titleFontSizeLG` | - |
| `@tabs-title-font-size-sm` | `titleFontSizeSM` | - |
```

**Review by MadCcc — APPROVED** [review]

**Pull request merged**